apps/gatsby/src/Sidebar.js

```diff
@@ -240,21 +234,88 @@ export default class Sidebar extends React.Component {
       <div className="extension">
         <div className="flexcontainer">
           {(previewWebhookUrl || webhookUrl) ?
-            <ExtensionUI
-              contentSlug={!contentSyncUrl && !!slug && slug}
-              previewUrl={previewUrl}
-              authToken={authToken}
-              onOpenPreviewButtonClick={async ({ previewWindow }) => {
-                await this.refreshPreview();
-                /**
-                 * ExtensionUI returns a reference to the opened tab (previewWindow) after eagerly
-                 * opening it with the given previewUrl. Because there is a small chance that this will
-                 * have a stale manifestId, we update the url in the opened preview tab just in case
-                 * to ensure that the user is redirected to the correct preview build
-                 */
-                previewWindow.location.href = this.getPreviewUrl();
-              }}
-            /> :
+
+            contentSyncUrl ? (
```

Comment:
I'm a little confused about this part. If we have a contentSyncUrl, that means we must have a content sync window open already and then we show this mocked button?

And if we don't have a content sync url then that means we just show the normal ExtensionUI button?